bundles/net.tourbook/src/net/tourbook/ui/views/WeatherProvidersUI.java

```diff
@@ -172,7 +171,15 @@ private void createUI_30_WeatherProviderMainPreferences() {
       GridDataFactory.fillDefaults().grab(true, true).applyTo(container);
       GridLayoutFactory.fillDefaults().applyTo(container);
       {
-         // Checkbox: Displays the full log
+         // Checkbox: Save the weather log in the tour weather's description
+         _chkSaveLogInTourWeatherDescription = new Button(container, SWT.CHECK);
+         _chkSaveLogInTourWeatherDescription.setText(
+               Messages.Pref_Weather_Check_SaveLogInTourWeatherDescription);
+         _chkSaveLogInTourWeatherDescription.setToolTipText(
+               Messages.Pref_Weather_Check_SaveLogInTourWeatherDescription_Tooltip);
+         GridDataFactory.fillDefaults().applyTo(_chkSaveLogInTourWeatherDescription);
+
+         // Checkbox: Display the full log
```

Comment:
Not for me because as I mentioned above, it requires me to scroll horizontally which takes time and is inefficient.
But I have unwrapped the lines for your convenience.

Reply:
I'm not sure it would work on my small LCD screen but I didn't know [this tool](http://brianapps.net/sizer/userguide.html)